core/services/pipeline/task.jsonparse.go

```diff
@@ -106,5 +109,21 @@ func (t *JSONParseTask) Run(_ context.Context, _ logger.Logger, vars Vars, input
 			return Result{Error: errors.Wrapf(ErrKeypathNotFound, `could not resolve path ["%v"] in %s`, strings.Join(path, `","`), data)}, runInfo
 		}
 	}
+
+	switch val := decoded.(type) {
+	case json.Number:
+		i64, err := val.Int64()
```

Comment:
Should we use a `big.Int` so that `uint256` is convered?

Reply:
Fixed